src/components/v5/common/ActionSidebar/partials/CreatedIn/CreatedIn.tsx

```diff
@@ -23,8 +23,7 @@ const CreatedIn: FC<CreatedInProps> = ({ filterOptionsFn, readonly }) => {
     name: DECISION_METHOD_FIELD_NAME,
   });
 
-  return decisionMethod === DecisionMethod.Reputation ||
-    decisionMethod === DecisionMethod.MultiSig ? (
+  return decisionMethod === DecisionMethod.Reputation ? (
```

Comment:
The createdIn field shouldn't always be there by default just because the decision method is multi-sig.